notes_app/client/app/notes/notes_service.js

```diff
@@ -5,14 +5,22 @@
 
   function notesService($http, $log) {
     var api_path = 'http://localhost:8000/api/v1/note/';
+    var data = {
+      notes: []
+    };
+    function updateList(list){
+      data.notes = list;
+    }
     return {
+      'data': data,
       'list': function(limit, offset) {
         return $http.get(api_path, {
           'params': {
             'limit': limit,
             'offset': offset
           }
         }).then(function successHandler(response) {
+          updateList(response.data.objects);
           return response.data;
```

Comment:
I'm not sure that you've got my comment. `return response.data;` - it doesn't look right to me.